bigframes/session/bq_caching_executor.py

```diff
@@ -373,6 +374,15 @@ def _cache_with_cluster_cols(
     ):
         """Executes the query and uses the resulting table to rewrite future executions."""
 
+        # Pseudocolumns can be queried and even written to anonymous query
+        # results tables, but they can't be materialized to an explicit
+        # destination table. Therefore, we rename the columns to all be writable
+        # before executing the SQL. See: b/405773140 for discussion about the
+        # _TABLE_SUFFIX pseudocolumn.
+        prev_col_ids = array_value.column_ids
+        new_col_ids, _ = utils.get_standardized_ids(prev_col_ids)
```

Comment:
In the case that standardization makes duplicate column names, we need another pass to remove duplicates.

Reply:
Nevermind, `get_standardized_ids` already does this. It's `label_to_identifier` that doesn't.